changelog.txt

```diff
@@ -38,6 +38,7 @@ Template for new versions:
 - `makeown`: halt any hostile jobs the unit may be engaged in, like kidnapping
 - `fix/loyaltycascade`: allow the fix to work on non-dwarven citizens
 - `control-panel`: fix setting numeric preferences from the commandline
+- `gui/confirm`: fix some confirm prompts not working
```

Comment:
no need for a changelog since there hasn't been a stable release with this broken yet